src/inference/Inference.jl

```diff
@@ -70,6 +71,9 @@ abstract type AdaptiveHamiltonian{AD} <: Hamiltonian{AD} end
 getchunksize(::Type{<:Hamiltonian{AD}}) where AD = getchunksize(AD)
 getADbackend(::Hamiltonian{AD}) where AD = AD()
 
+# Algorithm for sampling from the prior
+struct Prior end
```

Comment:
`Prior` can be viewed as the crudest way of performing approximate inference: 
```suggestion
struct Prior <: InferenceAlgorithm end
```